File: app/src/main/java/com/example/dpstuffproviderstore/models/ProductData.kt
package com.example.dpstuffproviderstore.models

import com.google.gson.annotations.SerializedName

data class ProductData (
        @SerializedName("id") var id : Int,
        @SerializedName("name") var name : String,
        @SerializedName("price") var price : Double?,
        @SerializedName("rating") var rating : Double?,
        @SerializedName("avail") var avail : Boolean?,
        @SerializedName("category") var category : String,
        @SerializedName("store") var store : String,
        @SerializedName("quantity") var quantity : Int? = 1
)

File: DPSP Api v2/Models/Data/ProductData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DPSPApiV2.Models.Data
{
    public class ProductData
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public float Price { get; set; }
        public float Rating { get; set; }
        public bool Avail { get; set; }
        public string Category { get; set; }
        public string Vendor { get; set; }
        public List<ProductImageData> Images { get; set; }
        public List<ProductAttributeData> Attributes { get; set; }
    }
}
